#!/usr/bin/env python3
# -*- coding: utf-8 -*-
PACKAGE = "itr_rpc"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("limit_velocity",	bool_t,		0, "Obey URDF joint velocity limits",	True)
gen.add("pos_x_err",		double_t,	0, "Position error x for IK solver",	.001,	0,	0.1)
gen.add("pos_y_err",		double_t,	0, "Position error y for IK solver",	.001,	0,	0.1)
gen.add("pos_z_err",		double_t,	0, "Position error z for IK solver",	.001,	0,	0.1)
gen.add("orient_x_err",		double_t,	0, "Orientation error x for IK solver",	6.5,	0,	7)
gen.add("orient_y_err",		double_t,	0, "Orientation error y for IK solver",	6.5,	0,	7)
gen.add("orient_z_err",		double_t,	0, "Orientation error z for IK solver",	6.5,	0,	7)

exit(gen.generate(PACKAGE, "itr_rpc", "Kinematics"))
